### ISSUE TITLE
Cannot zoom in. If I zoom in it gets rid of the text box and all other things

### ISSUE THREAD

**Issue by golfyplayz** [openscope/openscope#2113]
Browser: OperaGX

Airport: All I've tried

Approximate play time before issue: Not much

#### Issue Description
_please include any console errors and/or screenshots_ 

before zooming in - <img width="1094" alt="image" src="https://github.com/openscope/openscope/assets/146030843/667c8484-e23f-42bb-9d7d-94f705d83b06">

after zooming in - 
<img width="1078" alt="image" src="https://github.com/openscope/openscope/assets/146030843/e59ef522-68a6-4a81-b58e-7bd13171379b">

#### Steps to reproduce
-

**Comment by n8rzz** [comment]
We don't actively support `Opera` browsers. Things might work there, but if they do not we are not going to spend time fixing them. Opera represents less than 3% of modern browser usage.

**Comment by golfyplayz** [comment]
I've tried it on chrome as well and it does the same thing. I'm on laptop as well using a trackpad so that might be causing it too. I didn't fully read through the requirements so it may be because I'm using a trackpad.

**Comment by niin54** [comment]
Yes, I use trackpad on multiple browsers and I face the same issues.  I believe this does not occur when using a mouse scroll wheel, however I have not used one for a long time so it could well be mandela effect kicking in.  Nevertheless, I would assume the zooming problem with trackpad occurs due to the way the scope is modeled.  For now, my tip is to open the strip area, and just zoom in in steps, zooming in and then zooming out over the bottom bar or flight strips before zooming in again.  This has worked for me for quite a while.
